wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -34,7 +34,15 @@ <h1 class="w-header__title" id="header-title">
                 {% if description %}<div class="w-header__description">{{ description }}</div>{% endif %}
             </div>
             {% if search_url %}
-                <form class="col search-form" action="{% url search_url %}{% if query_parameters %}?{{ query_parameters }}{% endif %}" method="get" novalidate role="search">
+                <form
+                    class="col search-form"
+                    action="{% url search_url %}{% if query_parameters %}?{{ query_parameters }}{% endif %}"
+                    method="get"
+                    novalidate
+                    role="search"
+                    data-controller="w-search"
+                    data-action="{% verbatim %}change->w-search#search input->w-search#search{% endverbatim %}"
```

Comment:
Note that the `verbatim` is required here due to the HTML auto-formatter treating `>` inside data attributes as the end tag of the HTML element. Loveth - we can use this approach to work around the problem for now.

We will likely need to raise a bug to track this.
cc @thibaudcolas  / @Lovelyfin00

Reply:
This has been resolved now. I'll rebase this and remove the verbatim usage.